datalad/support/annexrepo.py

```diff
@@ -549,44 +552,72 @@ def get_file_key(self, file_):
             raise FileNotInAnnexError("Could not get a key for a file %s -- empty output" % file_)
         return entries[0]
 
+    @normalize_paths(map_filenames_back=True)
+    def find(self, files, batch=False):
+        """Provide annex info for file(s).
+
+        Parameters
+        ----------
+        files: list of str
+            files to find under annex
+        batch: bool, optional
+            initiate or continue with a batched run of annex find, instead of just
+            calling a single git annex find command
+
+        Returns
+        -------
+        list
+          list with filename if file found else empty string
+        """
+        objects = []
+        if batch:
+            objects = self._batched.get('find', path=self.path)(files)
+        else:
+            for f in files:
+                try:
+                    obj, er = self._run_annex_command('find', annex_options=[f], expect_fail=True)
+                    objects.append(obj)
+                except CommandError:
+                    objects.append('')
```

Comment:
I guess we could just store None's not just empty strings to make things more explicit and Pythonic